### DISCUSSION TITLE
After upgrading container in stacks under portainer, redis is not accessible.

### DISCUSSION THREAD

**Discussion by phantasy36** [docmost/docmost#396] [Help / Questions]
[_docmost-docmost-1_logs.txt](https://github.com/user-attachments/files/17350376/_docmost-docmost-1_logs.txt)

The stacks under portainer was successfully installed using the docker compose script and has been used normally. After upgrading under stacks in portainer on October 12, 2024, it cannot be opened and the above error is always prompted.

PS: All three containers are in the same container network: docmost_default

**Comment by Philipinho** [comment]
Is the redis container running?

**Comment by gomaaz** [comment]
Same problem here...Redis and DB is running.

**Reply by gomaaz** [reply]
ping on db and redis ist working. Error message persists.
